ui/src/components/integrationFormFields/integrationFormFields.tsx

```diff
@@ -23,6 +23,17 @@ export const IntegrationFormFields: FC<IntegrationFormFieldsInterface> = (props)
     });
   }, []);
 
+  const projectIdValidator = (value: string) => {
```

Comment:
Could we move this validator outside the component?

Or move it to a different file